base/common/python/pki/system.py

```diff
@@ -440,10 +459,18 @@ class SystemStatusClient(object):
     Client used to check the status of a Dogtag subsystem.
     """
 
-    def __init__(self, connection):
+    def __init__(self, connection, subsystem=None):
+
         self.connection = connection
 
-        self.get_status_url = '/admin/%s/getStatus' % connection.subsystem
+        if connection.subsystem is not None:
+            self.get_status_url = '/admin/%s/getStatus' % connection.subsystem
+
+        elif subsystem is None:
```

Comment:
(Nitpick) The reader might be confused between a `not None` and `None` checks. Usually we try to keep the exception/errors at the last.

For uniformity, how about?

        if connection.subsystem is not None:
            self.get_status_url = '/admin/%s/getStatus' % connection.subsystem
        elif subsystem is not None:
            self.get_status_url = '/%s/admin/%s/getStatus' % (subsystem, subsystem)
        else
            raise Exception('Missing subsystem for SystemStatusClient')

Reply:
OK, let me update it before pushing.